src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -4458,7 +4458,7 @@ string buildBindBack()
 						{
 							if (propertyPaths.properties.Length == 1)
 							{
-								var targetPropertyType = GetXBindPropertyPathType(propertyPaths.properties[0]).ToDisplayString(NullableFlowState.None);
```

Comment:
Don't we need to keep the nullableflowstate here?

Reply:
@jeromelaban The custom visitor implementation already ignores nullable annotations.